uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/resources/org/uberfire/ext/editor/commons/client/resources/i18n/Constants.properties

```diff
@@ -1,113 +1 @@
-ToggleCommentView.EnterComment=Enter Comment
-HtmlEditorView.CantSaveReadOnly=Can''t save, it''s on read only mode.
-HtmlEditorView.Loading=Loading...
-HtmlEditorView.Saving=Saving...
-HtmlEditorView.DiscardUnsavedData=Are you sure you want to discard unsaved data?
-HtmlEditorView.Ok=OK
-HtmlEditorView.Cancel=Cancel
-HtmlEditorView.Bold=Bold (Ctrl+B)
-HtmlEditorView.Italic=Italic (Ctrl+I)
-HtmlEditorView.Underline=Underline (Ctrl+U)
-HtmlEditorView.CreateLink=Create link
-HtmlEditorView.RemoveLink=Remove link
-HtmlEditorView.InsertImage=Insert image
-HtmlEditorView.InsertTable=Insert table
-HtmlEditorView.BigTitle=Big title
-HtmlEditorView.MediumTitle=Medium title
-HtmlEditorView.SmallTitle=Small title
-HtmlEditorView.Paragraph=Paragraph
-HtmlEditorView.Pre=Pre
-HtmlEditorView.PlainText=Plain text
-HtmlEditorView.Quote=Quote
-HtmlEditorView.Code=Code
-HtmlEditorView.FontSize=Font size
-HtmlEditorView.FontColor=Font color
-HtmlEditorView.BackgroundColor=Background color
-HtmlEditorView.UnorderedList=Unordered list
-HtmlEditorView.OrderedList=Ordered list
-HtmlEditorView.OutdentList=Outdent list
-HtmlEditorView.IndentList=Indent list
-HtmlEditorView.AlignLeft=Align left
-HtmlEditorView.AlignRight=Align right
-HtmlEditorView.AlignCenter=Align center
-HtmlEditorView.Undo=Undo
-HtmlEditorView.Redo=Redo
-HtmlEditorView.SwitchToHtmlView=Switch to HTML view
-HtmlEditorView.SwitchView=Switch view
-HtmlEditorView.TableOptions=Table options:
-HtmlEditorView.TableMergeCells=Merge cells
-HtmlEditorView.TableAddRowBefore=Add row before
-HtmlEditorView.TableAddRowAfter=Add row after
-HtmlEditorView.TableAddColumnBefore=Add column before
-HtmlEditorView.TableAddColumnAfter=Add column after
-HtmlEditorView.TableRemoveRow=Remove row
-HtmlEditorView.TableRemoveColumn=Remove column
-HtmlEditorView.FontSizeDialog=Font size:
-HtmlEditorView.BackgroundColorDialog=Background color:
-HtmlEditorView.FontColorDialog=Font color:
-HtmlEditorView.LinkDialog=Link:
-HtmlEditorView.ImageDialog=Image:
-HtmlEditorView.ImageAlignment=Alignment:
-HtmlEditorView.ImageAlignmentDefault=Default
-HtmlEditorView.ImageAlignmentLeft=Left
-HtmlEditorView.ImageAlignmentRight=Right
-HtmlEditorView.TableRows=Rows:
-HtmlEditorView.TableColumns=Columns:
-HtmlEditorView.xx-small=xx-small
-HtmlEditorView.x-small=x-small
-HtmlEditorView.small=small
-HtmlEditorView.medium=medium
-HtmlEditorView.large=large
-HtmlEditorView.x-large=x-large
-HtmlEditorView.xx-large=xx-large
-FileExport.Label=File exporting
-FileExport.PdfOrientation.Text=PDF orientation
-FileExport.PdfOrientation.Help=The PDF file orientation. Allowed values are PORTRAIT, LANDSCAPE
-FileExport.PdfUnit.Text=PDF units
-FileExport.PdfUnit.Help=The PDF file units. Allowed values are PT, MM, CM, IN
-FileExport.PdfFormat.Text=PDF page format
-FileExport.PdfFormat.Help=The PDF file page format. Allowed values are A[0-10], B[0-10], C[0-10]
\ No newline at end of file
+# Workaround for https://issues.jboss.org/browse/ERRAI-1101
```

Comment:
It's not clear what the workaround is (the issue description is loong :) )
Maybe add a comment for future maintainers like "English translation moved to Constants_default.properties as workaround to ..."